systems/crusher_quick_start_guide.rst

```diff
@@ -1702,6 +1702,21 @@ The aforementioned behavior is only true for FP atomicAdd() operations. Hardware
 In cases when contention is very low, a FP32 CAS loop implementing an atomicAdd() operation could be faster than an hardware FP32 LDS atomicAdd().
 Applications using single precision FP atomicAdd() are encouraged to experiment with the use of double precision to evaluate the trade-off between high atomicAdd() performance vs. potential lower occupancy due to higher LDS usage.
 
+------
+
+System Updates 
+============== 
+
+2022-12-29 
+---------- 
+
+On Thursday, December 29, 2022 the following system configuration settings will be updated on Crusher:
+
+ * Core specialization will be enabled: as a result, system processes will be constrained to core 0 on every node.
```

Comment:
> system processes will be constrained to core 0 on every node

With `-S 8`, wouldn't the processes be constrained to the first core in each NUMA (depending on distribution)? Specifying core 0 feels like it's just core 0 every time, potentially.

Reply:
Thank you @hagertnl! I made some edits to make it clearer (hopefully).